### DISCUSSION TITLE
HStack and VStack

### DISCUSSION THREAD

**Discussion by taneba** [mantinedev/mantine#1200] [Feature requests]
Hi team. Thank you for building this great library.

I've been looking for an alternative to chakra ui, and found this great library.
I'm wondering why Stack component doesn't support horizontal direction, while those ui libraries like chakra, native base, mui supports VStack/HStack (or XStack/VStack), which is really good.

Do you have a plan to support something like that?
If not, what is the recommended way to implement horizontal stack layout?

**Comment by rtivital** [comment]
https://mantine.dev/core/group/

**Reply by taneba** [reply]
Wow, thank you for your quick reply!
